features/specs/gezag-minderjarige/niet-ouder-gezag.feature

```diff
@@ -0,0 +1,218 @@
+#language: nl
+
+Functionaliteit: in verschillende situaties heeft een niet ouder (ook) het gezag over een minderjarige
+
+  Een `niet ouder` is iemand die gezag heeft over een minderjarige maar niet de ouder is.
+  Dat wil zeggen, de persoon is:
+  - niet de moeder uit wie het kind geboren is
+  - niet iemand die het kind erkend heeft
+  - niet iemand die het kind geadopteerd heeft
+  Een `niet ouder` kan een vader zijn, maar ook een oma, opa, peetoom of andere relatie.
```

Comment:
Deze opsomming van soorten niet-ouder is denk ik vooral verwarrend.
Een vader is normaal gesproken een ouder, is alleen juridisch een niet-ouder, wanneer de biologische vader het kind niet erkend heeft. Maar dan is de vader ook geen niet-ouder in de zin van derde of voogd zoals in de regels en scenario's hieronder beschreven.

Oma, opa en peetoom kunnen niet-ouder zijn voor gezag, maar alleen na gerechtelijke uitspraak (D, 1D of 2D).

De categorie "andere relatie" bevat hele belangrijke groepen die in de regels en scenario's wel worden geïllustreerd:
- een partner
- een ex-partner, ook als de moeder inmiddels gehuwd is met een andere partner (wordt helaas niet geïllustreerd in de scenario's)
- een voogdijinstelling